modules/virt-adding-templates-to-custom-namespace.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-adding-templates-to-custom-namespace.adoc
+
+[id="virt-adding-templates-to-custom-namespace_{context}"]
+= Adding templates to a custom namespace
+
+The `ssp-operator` deploys Red Hat templates to the `openshift` namespace by default. Templates in the `openshift` namespace are publicly availably to all users. To add templates to a custom namespace, edit the `hco` custom resource (CR) which contains the `ssp-operator`.
+
+.Procedure
+
+* View the list of Red Hat templates that are available in the `openshift` namespace.
++
+[source,terminal]
+----
+$ oc get templates -n openshift
+----
++
+* Open the `hco` custom resource (CR).
++
+[source,terminal]
+----
+$ oc edit hco -n openshift -cnv kubevirt-hyperconverged
+----
++
+* Find the `commonTemplatesNamespace` attribute and specify the custom namespace. Example:
```

Comment:
`spec.commonTemplatesNamespace` may not exist in the CR initially

Reply:
Updated the content to reflect that user adds this attribute.